go.mod

```diff
@@ -858,7 +817,7 @@ replace k8s.io/cri-api => k8s.io/cri-api v0.25.5
 // Use custom Trivy fork to reduce binary size
 // Pull in replacements needed by upstream Trivy
 replace (
-	github.com/aquasecurity/trivy => github.com/DataDog/trivy v0.0.0-20240326085931-6ae645e90ebc
+	github.com/aquasecurity/trivy => github.com/DataDog/trivy v0.0.0-20240326163220-37861582e30d
```

Comment:
This isn't merged yet right ?

Reply:
I'm afraid this will be very hard to maintain :-( Should it be handled by @DataDog/agent-shared-components ?

Reply:
Yes, it's going to be a pain to maintain, because that cleaning work would have to be re-done each time we rebase on the upstream Trivy.